org.eclipse.scanning.event/src/org/eclipse/scanning/event/ConsumerImpl.java

```diff
@@ -429,6 +443,15 @@ public void run() throws EventException {
 		PauseBean pbean = getPauseBean(getSubmitQueueName());
 		if (pbean!=null) processPause(pbean); // Might set the pause lock and block on checkPaused().
 		 
+		// We should pause if there are things in the queue
+		// This is because on a server restart the user will 
+		// need to choose the visit again and get the baton.
+		// NOTE: Not all consumers check the submit queue and
+		// pause before they start.
+        checkStartPaused();
+		
+		// It is possible to call start() and then awaitStart().
+		if (latchStart!=null) latchStart.countDown();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00122)